src/component/homepage/Home.jsx

```diff
@@ -1,9 +1,19 @@
-import React from 'react'
+import React, { useEffect, useState } from 'react'
 import Card from './Card'
+import axios from 'axios'
 function Home() {
+  const [article, Setarticle] = useState([])
+  const url = 'https://mynew-nc-news.onrender.com/api/articles'
+useEffect(()=>{
+axios.get(url)
+.then((res)=>{
+Setarticle(res.data.article);
+})
+},[])
+
```

Comment:
This works, but I would recommend extracting this out and making use of an axios's instance as you will be creating a lot of requests for the same base url.

Notes for this if you are unsure how it works:
l2c.northcoders.com/courses/sd-notes/front-end/#sectionId=axios,step=creating-an-axios-instance